File: 0344-reverse-string/0344-reverse-string.c
void reverseString(char* s, int n)
{
  int i,j,c;

 j=n-1;
 for(i=0;i<n/2;i++)
 {
   c=s[i];
   s[i]=s[j];
   s[j]=c;
   j--;
 }
 

}

File: 0344-reverse-string/0344-reverse-string.java
class Solution {
    public void reverseString(char[] s) {
        int n = s.length;
        char rev[] = new char[n];
        for(int i=0; i<n; i++)
          rev[i] = s[n-i-1];

        for(int i=0; i<n; i++)
          s[i] = rev[i];
    }
}